source/_integrations/fmi.markdown

```diff
@@ -0,0 +1,154 @@
+---
+title: FMI
+description: Instructions on how to integrate FMI within Home Assistant.
+ha_category:
+  - Weather
+  - Sensor
+ha_release: 0.108.3
```

Comment:
```suggestion
ha_release: 0.109
```

Reply:
Working on it. Plan is to get it in by tomorrow.

Reply:
I changed the PR to draft. Will mark it for review once everything is in place. Sorry for wasting your time!

Reply:
Core PR link - https://github.com/home-assistant/core/pull/34213